islandora_solr.install

```diff
@@ -234,6 +234,54 @@ function islandora_solr_update_7001() {
   db_create_table('islandora_solr_collection_sort_strings', $schema['islandora_solr_collection_sort_strings']);
 }
 
+/**
+ * Fix the table islandora_solr_collection_sort_strings, if possible.
+ */
+function islandora_solr_update_7002() {
+  module_load_include('inc', 'islandora', 'includes/utilities');
+  $schema = islandora_solr_schema();
+  $results = db_select('islandora_solr_collection_sort_strings', 'c')
+    ->fields('c', array('collection_pid', 'sort_string'))
+    ->execute();
+
+  $is_empty = function($o) {
+    return (empty($o) || trim($o) == FALSE);
```

Comment:
The `trim($o) == FALSE` is... less than expected? I know it works in PHP, but comparing a string directly against a boolean feels weird. How would we feel about something like `!trim($o)` (in my mind, it implies something of a cast, which is more understandable than PHP's type juggling... maybe? Dunno...)?